recipes/indexed_bzip2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "indexed_bzip2" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3b02216a5c49d228595afed418b978de5b7bed314e594ee60a75b1a91a9431ce
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - ibzip2 = indexed_bzip2:cli
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - python
+    - pip
+    - cython
+  run:
+    - python
+
+test:
+  imports:
+    - indexed_bzip2
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/mxmlnkn/indexed_bzip2
+  summary: 'Fast random access to bzip2 files'
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
This project contains external source that is being statically linked. You must include the license for that code here. i.e. It's revending CXXOPTS, so add the license for CXXOPTS.

https://github.com/conda-forge/cxxopts-feedstock

There are other external libraries revended in the upstream repo, but don't think they are used for this python API.

Reply:
> There are other external libraries revended in the upstream repo, but don't think they are used for this python API.

Yes, `bzip2-tests` is only used for tests and `zlib` only for `pragzip`. Both are not built/linked with `indexed_bzip2`. So the only linked dependency is cxxopts.

`indexed_bzip2` uses modified [0-BSD code](https://github.com/landley/toybox/blob/c77b66455762f42bb824c1aa8cc60e7f4d44bdab/toys/other/bzcat.c) directly. While we are at it, I think even if not required, it could be mentioned:

https://github.com/landley/toybox/

As far as I read, these should be added into a `THIRDPARTY.yml`.

Reply:
Sorry for the delay.

I added both the correct version of the cxxopts license and a reference to the toybox license.
The Python source distribution doesn't ship either of the two, so I opted for including them in `THIRDPARTY.yml` next to the recipe's `meta.yaml`

Cheers,
Andreas

Reply:
Thanks for all your work.